src/main/data/documents.json

```diff
@@ -3676,18 +3792,70 @@
         "publicationDate": "1998-11-01T07:00:00.000Z",
         "details": "Cannot locate supersededBy doc"
     },
+    {
+        "docId": "bt601-4-199510",
+        "publisher": "ITU-R",
+        "type": "Recommendation",
+        "label": "BT.601.4 (10/95)",
+        "title": "Studio encoding parameters of digital television for standard 4:3 and wide screen 16:9 aspect ratios",
+        "superseded": true,
+        "supersededBy": [
+            "bt601-5-199510"
+        ],
+        "url": "https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.601-4-199407-S!!MSW-E.doc",
+        "publicationDate": "1994-07-30T07:00:00.000Z"
+    },    
     {
         "docId": "bt601-5-199510",
         "publisher": "ITU-R",
         "type": "Recommendation",
         "label": "BT.601.5 (10/95)",
         "title": "Studio encoding parameters of digital television for standard 4:3 and wide screen 16:9 aspect ratios",
-        "status": "Superseded",
```

Comment:
Please keep `"status": "Superseded",` here until status logic is redone.

Reply:
Done. See commit [c69259c29caa1e051578c810701e63be0ffba231](https://github.com/SteveLLamb/mediastandards-registry/pull/9/commits/c69259c29caa1e051578c810701e63be0ffba231)